apps/docs/content/components/Grid.mdx

```diff
@@ -0,0 +1,274 @@
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+    </Grid>
+  </>
+  <>
+    <Text as="p" variant="muted">
+      Spacious:
+    </Text>
+    <Grid
+      gap="spacious"
+      enableOverlay
+      style={{backgroundColor: 'var(--base-color-scale-red-1)'}}
+    >
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+      <Grid.Column span={1}></Grid.Column>
+    </Grid>
+  </>
+</Stack>
+```
+
+### Grid gutters
+
+Use `gutters` to enable horizontal spacing on the outside of the `Grid`.
+
+```jsx live
+<Stack direction="vertical">
+  <>
+    <Text as="p" variant="muted">
+      Default (none):
+    </Text>
+    <Grid
+      enableOverlay
+      style={{backgroundColor: 'var(--base-color-scale-red-1)'}}
+    >
+      <Grid.Column></Grid.Column>
+    </Grid>
+  </>
+  <>
+    <Text as="p" variant="muted">
+      Condensed:
+    </Text>
+    <Grid
+      gutters="condensed"
+      enableOverlay
+      style={{backgroundColor: 'var(--base-color-scale-red-1)'}}
+    >
+      <Grid.Column></Grid.Column>
+    </Grid>
+  </>
+  <>
+    <Text as="p" variant="muted">
+      Normal:
+    </Text>
+    <Grid
+      gutters="normal"
+      enableOverlay
+      style={{backgroundColor: 'var(--base-color-scale-red-1)'}}
+    >
+      <Grid.Column></Grid.Column>
+    </Grid>
+  </>
+  <>
+    <Text as="p" variant="muted">
+      Spacious:
+    </Text>
+    <Grid
+      gutters="spacious"
+      enableOverlay
+      style={{backgroundColor: 'var(--base-color-scale-red-1)'}}
+    >
+      <Grid.Column></Grid.Column>
+    </Grid>
+  </>
+</Stack>
+```
+
+## Component props
+
+### Grid <Label>Required</Label>
+
+| name            | type                                                                             | default    | required | description                                           |
+| --------------- | -------------------------------------------------------------------------------- | ---------- | -------- | ----------------------------------------------------- |
+| `as`            | <PropTableValues values={['div', 'section', 'span']} addLineBreaks />            | `'div'`    | `false`  | The HTML element used to render the grid.             |
```

Comment:
Should we also add `ul` to this list? Which makes the columns as `li`?

Reply:
Is that a use-case you run into often @enstyled? 🤔 I wonder if there's an accessibility element to using list markup for this, which could result in unintentionally verbose callouts if we did it. cc. @TylerJDev.

Reply:
I would not say I run into it often, but for semantic reasons, we might want to display a 3-4 items list in a responsive grid. Interested in hearing about the accessibility details.

Reply:
> we might want to display a 3-4 items list in a responsive grid

I'm slightly struggling to visualise this, do you have an example I can refer to?

I think a [Stack](https://primer.style/brand/components/Stack/) in responsive configuration could suit this use-case better.

Reply:
Taking a look at it, having `<ul>` as an option might cause some unintended use. Looking at the `Stack` component mentioned by @rezrah, I think that might be a better choice when needing to utilize a list. I'd say, if we truly needed a list with this component then we could utilize ARIA roles (e.g. `role="list"` and `role="listitem"`), as these roles would convey the list to assistive technologies (screen readers, etc). 

I might not be thinking of all scenarios though, I'm only imagining this grid as primarily handling layout, so correct me if I'm wrong! :grin:

Reply:
Yes, perhaps a Stack might be better suited for this case 👍🏼